### ISSUE TITLE
Unable to use /@dnd-kit/core/dist/components/Accessibility/Accessibility.d.ts(2,13): '=' expected.  TS1005

### ISSUE THREAD

**Issue by thekennysong** [clauderic/dnd-kit#370]
I have not used this repo before and after a fresh install I am running into a Typescript issue on build. I have all the monorepos installed via yarn.

![image](https://user-images.githubusercontent.com/5256555/126379582-0e7580c3-6fc1-4075-ac12-34f10e923ac1.png)

**Comment by clauderic** [comment]
You're likely using an old version of TypeScript, make sure the resolved TypeScript version is `^4.0.5`
